.github/workflows/2-system.yaml

```diff
@@ -29,7 +29,7 @@ on:
 env:
   # openwhisk env
   TEST_SUITE: System
-  ANSIBLE_CMD: "ansible-playbook -i environments/local -e docker_image_prefix=testing"
+  ANSIBLE_CMD: "ansible-playbook -i environments/local -e docker_image_prefix=testing -e container_pool_akka_client=false"
```

Comment:
The akka client is not well aligned with the FunctionPollingContainerProxy.
So I changed this to the traditional apache HTTP client.